### PR TITLE
Update preference view to show list of devfile registries

### PR DESCRIPTION
**Pull request by valaparthvi**
**What type of PR is this:**
/kind feature

**What does this PR do / why we need it:**
This PR moves `odo preference registry list` to `odo preference view`.
**Which issue(s) this PR fixes:**

Fixes part of #5535 

**PR acceptance criteria:**

- [x] Unit test 

- [x] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**
Example 1-
```sh
$ odo preference view

Preference parameters:
 PARAMETER           VALUE          
 ConsentTelemetry    true           
 Ephemeral                          
 PushTimeout                        
 RegistryCacheTime                  
 Timeout             5              
 UpdateNotification  true (default) 

Devfile registries:
 NAME                    URL                          SECURE 
 DefaultDevfileRegistry  https://registry.devfile.io  No   
```

Example 2 - 
```sh
$ odo preference view
Preference parameters:
 PARAMETER           VALUE          
 ConsentTelemetry    true           
 Ephemeral                          
 PushTimeout                        
 RegistryCacheTime                  
 Timeout             5              
 UpdateNotification  true (default) 

Devfile registries:
 ⚠  No devfile registries added to the configuration. Refer `odo preference registry add -h` to add one
```

### CODE REVIEW

**Review by rm3l — CHANGES_REQUESTED** [review]

File: pkg/odo/cli/preference/view.go [hunk 1]
```diff
@@ -47,19 +50,59 @@ func (o *ViewOptions) Validate() (err error) {
...
 }
 
+func HumanReadableOutput(preferenceList preference.PreferenceList, registryList *[]preference.Registry) {
+	tStyle := table.Style{
+		Box: table.BoxStyle{PaddingLeft: " ", PaddingRight: " "},
+		Color: table.ColorOptions{
+			Header: text.Colors{text.FgHiGreen},
+		},
+	}
+	preferenceT := table.NewWriter()
```
[the remark this answers is not in the document]
Comment (valaparthvi) [reply pkg/odo/cli/preference/view.go:66]: Oh yes, definitely. I didn't realize that was in.

File: pkg/odo/cli/preference/view.go [hunk 2]
```diff
@@ -47,19 +50,49 @@ func (o *ViewOptions) Validate() (err error) {
...
+			secure = "Yes"
+		}
+		registryT.AppendRow(table.Row{registry.Name, registry.URL, secure})
+	}
+
+	log.Info("Preference parameters:")
+	preferenceT.Render()
+	log.Info("\nDevfile registries:")
+	if registryList == nil || len(*registryList) == 0 {
+		log.Warning("No devfile registries added to the configuration. Refer `odo preference registry add -h` to add one")
```
Comment (rm3l) [inline pkg/odo/cli/preference/view.go:91]:
```suggestion
		log.Warning("No devfile registries added to the configuration. Refer to `odo preference registry add -h` to add one")
```

**Review by rm3l — CHANGES_REQUESTED** [review]
Just one last small change ^^

**Review by rm3l — APPROVED** [review]

**Review by rm3l — APPROVED** [review]

**Pull request merged**